packages/bitcore-node/src/routes/api/fee.ts

```diff
@@ -1,22 +1,45 @@
 import { Request, Response } from 'express';
+import config from '../../config';
+import logger from '../../logger';
 import { ChainStateProvider } from '../../providers/chain-state';
+import { IUtxoNetworkConfig } from '../../types/Config';
 import { CacheTimes } from '../middleware';
 import { CacheMiddleware } from '../middleware';
 const router = require('express').Router({ mergeParams: true });
 const feeCache = {};
 
+const feeModes = {
+  BTC: ['CONSERVATIVE', 'ECONOMICAL'],
+  LTC: ['CONSERVATIVE', 'ECONOMICAL']
+};
+
 router.get('/:target', CacheMiddleware(CacheTimes.Second), async (req: Request, res: Response) => {
   let { target, chain, network } = req.params;
+  let { mode } = req.query as any;
```

Comment:
Specifying 'any' here is redundant as req.query is already any. Not sure if you included for some linting error but it works fine for me.

Reply:
removing `as any` implicitly sets mode to a string which doesn't jive with the `'CONSERVATIVE' | 'ECONOMICAL' | undefined` type in the rest of the code.

Reply:
After a closer look, `let { mode} = req.query;` sets mode to any. I had likely been seeing an error that was left in my linter after moving mode from req.params to req.query.